doc/whats_new/v0.21.rst

```diff
@@ -43,6 +43,15 @@ Support for Python 3.4 and below has been officially dropped.
     section should be ordered according to the label ordering above. Entries
     should end with: :issue:`123456` by :user:`Joe Bloggs <joeongithub>`.
 
+:mod:`sklearn.base`
+...................
+
+- |API| The default value of ``multioutput`` in the ``score`` method of
```

Comment:
Reword this "the R2 score used when calling ``score`` on a regressor will use multioutput='uniform_average' from version ..."